python/dgl/graphbolt/__init__.py

```diff
@@ -1,3 +1,4 @@
 """GraphBolt"""
 
+from .graph_storage import *
```

Comment:
does `graph_storage` exist?

Reply:
For later PR usage

Reply:
then it's better to add this at that time.

Reply:
Both would be ok, according to the comment in #5700 , we move the `graphbolt` level changes in this PR.

Reply:
Only add code when the code is valid, if the graph_storage does not even exist, why import?

Reply:
Make sense, have dropped the changes.